### PR TITLE
Fixes issue #23

### LINKED ISSUE #23: [Doc] Fix Screenshots
## Problem Description

Screenshots look messy, require refining. Steps to follow:
- [x] Modify screenshots (base screenshots provided below)
- [x] Replace the following code:
  ```
  ![image](https://user-images.githubusercontent.com/61582763/106519739-c508e800-6501-11eb-9133-1de5952cfe40.png)
  ![image](https://user-images.githubusercontent.com/61582763/106519960-16b17280-6502-11eb-94eb-81fd3fabfd9e.png)
  ```
  with the new image code. (template below:)

    ```
  <img src="new-url" alt="alt-text" width=50%>
  <img src="new-url" alt="alt-text" width=50%>
  ```
- [x] Add `alt` and `width` attributes as per your discretion

## Base Images
<details>
<summary><b>Note:</b></summary>
<ol>
<li>These are the preferred base images, please feel free to take new ones.</li>
<li>Kindly double check if they are legible</li>
<li>You can refer to the original at <a href="https://github.com/Akshu-on-github/Basic-Programs-Python/blob/master/CONTRIBUTING.md/#adding-sample-code">
<ol>
</details>

### PR DESCRIPTION
**Pull request by kritigupta45**
This PR fixes the issue by replacing the screenshots with the following code-

```
<img src="https://user-images.githubusercontent.com/82928579/119253135-69a45800-bbcd-11eb-824a-f08b1e549024.PNG" alt="issues & PRs" width=50%>
<img src="https://user-images.githubusercontent.com/82928579/119253194-bb4ce280-bbcd-11eb-8db2-8ca5abd18c23.PNG" alt="open issues" width=70%>
```
newcapture1
newcapture2

also, changed the sample code template with-

```
<img src="https://user-images.githubusercontent.com/82928579/119253831-af165480-bbd0-11eb-86ee-f359da5ea9c2.png" alt="sample code" width=50%>
```
newcapture3

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,13 +10,14 @@
...
+
   - If the resemblance is minimal, please reference the previous PR/issue
   - If there's a large overlap, please start a discussion and ping a mod for clarification
 - Raise an issue that briefly explains the Sample Code
   - Add references as and when possible
   - **Plagiarism is intellectual theft and is severely condemned.**
 - While opening the PR with the appropriate changes, reference the relavant issue
   - Please use the `Sample Code` template
-  ![image](https://user-images.githubusercontent.com/61582763/106519444-4c099080-6501-11eb-8ca9-1b93a55ee813.png)
+<img src="https://user-images.githubusercontent.com/82928579/119253831-af165480-bbd0-11eb-86ee-f359da5ea9c2.png" alt="sample code" width=50%>
```
Comment (Akshu-on-github) [inline CONTRIBUTING.md]: Please give verbal directions to identify the emphasized elements
For example,
```html
<!-- For the Issue and PR tabs-->
<img src="..." alt="Click the tabs next to the 'Code' tab (found below the name of the repository)">
```

**Review by Akshu-on-github — CHANGES_REQUESTED** [review]
Thanks for contributing - the screenshots look so much nicer!
Just that one thing about the alt text - I hope you don't mind!

**Comment by kritigupta45** [comment]
Apologies! the following changes have been made as per the instructions, kindly have a look.
- Line 13 & 23 respectively:

```
<img src="https://user-images.githubusercontent.com/82928579/119253135-69a45800-bbcd-11eb-824a-f08b1e549024.PNG" alt="Click the tab 'Issues' or 'Pull requests' next to the 'code' tab (below the name of the repository)" width=50%> 
<img src="https://user-images.githubusercontent.com/82928579/119253831-af165480-bbd0-11eb-86ee-f359da5ea9c2.png" alt="Use the sample code template- click on 'Get started' (green button)" width=50%>
```

**Comment by Akshu-on-github** [comment]
Thanks a million @imkritigupta! And congrats on your successful contribution!

**Pull request merged**